app/assets/javascripts/discourse/app/components/user-preferences/user-passkeys.gjs

```diff
@@ -111,6 +102,8 @@ export default class UserPasskeys extends Component {
         bodyComponentModel: registrationResponse,
       });
     } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error(error);
```

Comment:
Leftover or (considering the eslint-disable...) intentional?

Reply:
Thanks, it's intentional. In this particular case, it's good to output the error in the console, it can help debug issues with a particular credential.